content/sensu-core/1.7/reference/clients.md

```diff
@@ -108,6 +108,12 @@ _WARNING: registration events are not stored in the event registry (in the Sensu
 [data store][8]), so they are not accessible via the Sensu API; however, all
 registration events are logged in the [Sensu server][5] log._
 
+#### Deregistration events
+
+Similarly to registration events, Sensu can create and process a deregistration event when the Sensu client process stops.
```

Comment:
```suggestion
Similar to registration events, Sensu can create and process a deregistration event when the Sensu client process stops.
```

Reply:
"Similarly" is correct here because it's referring to "can create and process", not "Sensu" or "deregistration events".